.github/workflows/validate_pr_title.yml

```diff
@@ -0,0 +1,26 @@
+name: Validate PR Conventional Commit
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, edited]
+
+jobs:
+  validate-pr-title:
+    runs-on: ubuntu-latest
+    steps:
+      - name: PR Conventional Commit Validation
+        uses:  ytanikin/PRConventionalCommits@1.1.0
+        with:
+         task_types: '["fix","feat","refactor","spike","hotfix","revert","chore","docs","test","build"]'
```

Comment:
Missing `ci`?

Reply:
We haven't included those in our guide in Notion, but we could. IIRC, we thought `ci` didn't add much that `build` doesn't already cover, but `perf` could be an interesting one to add.